app/scripts/lib/app-start.js

```diff
@@ -406,6 +410,25 @@ function (
       }
     },
 
+    initializeHeightObserver: function () {
+      var self = this;
+      if (self._isInAnIframe()) {
+        var heightObserver = new HeightObserver({
+          target: self._window.document.body,
+          window: self._window
+        });
+
+        heightObserver.on('change', function (height) {
+          self._iframeChannel.send('height', { height: height });
```

Comment:
The `height` event needs added to the protocol doc. @jpetto - you should expect this!